website/pages/docs/components/transitions.mdx

```diff
@@ -166,6 +166,31 @@ function CollapseEx() {
 }
 ```
 
+### Single Way Transition
+
+```jsx
+function SingleWay() {
```

Comment:
![image](https://user-images.githubusercontent.com/18474538/105373170-261ceb80-5be5-11eb-80d1-94e4f1d1723d.png)
how test it?

Reply:
You can't it will just open on the page load.
Here's a short video example demonstration


https://user-images.githubusercontent.com/56165400/105379821-a879cf80-5c26-11eb-810e-ba43a3597409.mp4

Reply:
> You can't it will just open on the page load.
> Here's a short video example demonstration
> screen1.2.mp4

Now I see!

But I still think that you should find a way to represent this effect, in documentation!
I mean, still is hard to someone else to understand your PR without see this video

Reply:
Yeah I definitely agree with you. Maybe we should add something like setTimeout when we get to that example?

Reply:
> Yeah I definitely agree with you. Maybe we should add something like `setTimeout` when we get to that example? Or maybe add this video to the example... Maybe we could add detailed description for it

Hmmm the timeout approach sounds good! I Agree with it. Quite simple and efficient.

Reply:
Sounds good!

Reply:
Hey, so will you merge this pull request?